packages/roosterjs-editor-dom/lib/table/VTable.ts

```diff
@@ -90,12 +110,17 @@ export default class VTable {
                     }
                 });
             });
+            const formatInfo = getTableFormatInfo(this.table);

```

Comment:
getTableFormatInfo() should be called when create VTable, and then save format in writeBack(). Then during the whole lifecycle of VTable we know its current format

Reply:
And if the format is still the default value when save, no need to save it. Any existing format can be removed. This can happen for a table never be formatted